packages/one-app-bundler/bin/postProcessOneAppBundle.js

```diff
@@ -18,6 +18,8 @@ const path = require('path');
 const fs = require('fs');
 const { hashElement } = require('folder-hash');
 const readPkgUp = require('read-pkg-up');
+
```

Comment:
These changes should not be required as this is does not affect module builds. The development build of one app should only be used when working on the One App server itself, not when working on modules.